tools/common/Target.cs

```diff
@@ -222,6 +222,7 @@ public void GatherFrameworks ()
 						case "Metal":
 						case "MetalKit":
 						case "MetalPerformanceShaders":
+						case "CHIP":
```

Comment:
@dalexsoto out of curiosity, why doesn't the default case handle this?

https://github.com/xamarin/xamarin-macios/blob/401befe28f90698ba5306b7fd320a2bddf9001b9/tools/common/Target.cs#L251-L258

Reply:
@rolfbjarne good question, I did not have a close look at it I wonder if there is a casing issue somewhere @mandel-macaque looks like we'll need to check deeper

Reply:
I'm looking atm.